test/02_mb_dispatch/Makefile

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2022, Xilinx Inc.
```

Comment:
This checks in a bunch of makefiles but the tests are built and run via llvm-lit. We could have makefiles if we refactored the lit infrastructure to build via makefiles. Otherwise the makefiles are untested and get out of sync and we shouldn't have them.

Reply:
Didn't realize that. I can delete them, some of the tests like 13 and 21 do have Makefiles idk if we want to do anything with those.

Reply:
Removed all added Makefiles in [e2cdfad](https://github.com/Xilinx/mlir-air/pull/185/commits/e2cdfade011aaa08ef6d4ac2765de032d8a40f03).